Quick-Start-Guide/Quickstart.md

```diff
@@ -139,17 +155,18 @@ You should see something like this (the Volume ID will differ):
                 Volume ID: f25cc3d8-631f-41bd-96e1-3e22a4c6f71f
                 Status: Started
                 Snapshot Count: 0
-                Number of Bricks: 1 x 2 = 2
+                Number of Bricks: 1 x 3 = 3
                 Transport-type: tcp
                 Bricks:
                 Brick1: server1:/data/brick1/gv0
                 Brick2: server2:/data/brick1/gv0
+                Brick3: server3:/data/brick1/gv0
                 Options Reconfigured:
                 transport.address-family: inet
 
 
 Note: If the volume is not started, clues as to what went wrong will be
```

Comment:
If the volume does not show "Started", the files under /var/log/glusterfs/glusterd.log should be checked in order to debug and diagnose the situation. These logs can be looked at on one or, all the servers configured.